src/lib/abstractsocialpostcachedatabase.cpp

```diff
@@ -347,7 +347,7 @@ bool AbstractSocialPostCacheDatabase::read()
                 "SELECT identifier, name, body, timestamp "
                 "FROM posts");
     if (!d->accountIdFilter.isEmpty()) {
-        postQueryString += " WHERE identifier IN (" + filteredPostIds.join(',') + ')';
+        postQueryString += " WHERE identifier IN (\"" + filteredPostIds.join("\",\"") + "\")";
     }
```

Comment:
Account ids are integers (sorry, should have said "post identifiers are strings" in description) so it is fine as is.